test/e2e/tests/test_subscription_deletion_revokes_keys.py

```
@@ -0,0 +1,134 @@
+            })
+            _wait_reconcile()
+
+            _delete_cr("maassubscription", sub_name, namespace=MODEL_NAMESPACE)
+            _wait_reconcile()
+            log.info("Subscription with no keys deleted successfully ✓")
+
+        except Exception as e:
+            log.error(f"Deletion failed: {e}")
+            try:
+                _delete_cr("maassubscription", sub_name, namespace=MODEL_NAMESPACE)
+            except:
```

Ruff (0.15.20)

[error] Do not use bare `except`

(E722)